rules/crowdstrike_rules/crowdstrike_detection_passthrough.yml

```diff
@@ -63,3 +64,76 @@ Tests:
           "eid": 118,
           "PatternDispositionFlags": "{\n  \"BlockingUnsupportedOrDisabled\": false,\n  \"BootupSafeguardEnabled\": false,\n  \"CriticalProcessDisabled\": false,\n  \"Detect\": false,\n  \"FsOperationBlocked\": false,\n  \"HandleOperationDowngraded\": false,\n  \"InddetMask\": false,\n  \"Indicator\": false,\n  \"KillActionFailed\": false,\n  \"KillParent\": false,\n  \"KillProcess\": false,\n  \"KillSubProcess\": false,\n  \"OperationBlocked\": false,\n  \"PolicyDisabled\": false,\n  \"ProcessBlocked\": true,\n  \"QuarantineFile\": true,\n  \"QuarantineMachine\": false,\n  \"RegistryOperationBlocked\": false,\n  \"Rooting\": false,\n  \"SensorOnly\": false,\n  \"SuspendParent\": false,\n  \"SuspendProcess\": false\n}",
         }
+  -
+    Name: Low Severity Finding (FDREvent)
+    ExpectedResult: true
+    Log:
+      {
+        "cid": "11111111111111111111111111111111",
+        "Technique": "PUP",
+        "ProcessId": 377077835340488700,
+        "AgentIdString": "00000000000000000000000000000000",
+        "DetectName": "NGAV",
+        "ComputerName": "macbook",
+        "ProcessStartTime": "2021-09-18 20:38:51Z",
+        "GrandparentCommandLine": "/sbin/launchd",
+        "MACAddress": "aa-00-00-00-00-00",
+        "CommandLine": "/Applications/app.app/Contents/MacOS/pup app",
+        "Objective": "Falcon Detection Method",
+        "Nonce": 1,
+        "SHA256String": "3333333333333333333333333333333333333333333333333333333333333333",
+        "event": {
+          "ExternalApiType": "Event_DetectionSummaryEvent"
+        },
+        "fdr_event_type": "Event_DetectionSummaryEvent",
+        "PatternDispositionValue": 2176,
```

Comment:
Should we have a processed `Crowdstrike.FDREvent` here?

Reply:
How about this one too?

Reply:
Yup, I haven't forgotten. Just didn't tend to it yet 😄 
Done now.
Changed only the one that I added that passes. I didn't touch the old one above.
https://github.com/panther-labs/panther-analysis/pull/648/commits/d9bd49f24e8a656741eac67be3916ade06c0b053

Reply:
Resolving as it's outdated.